components/debug/debug-dialog.tsx

```
@@ -1,83 +1,153 @@
+                      disabled={!debugData?.logsUrl}
+                    >
+                      <Download className="mr-2 h-4 w-4" />
+                      Download Debug Logs
+                    </Button>
+                  </div>
+                  <DebugViewer html={debugData.html} />
+                </div>
+              ) : (
+                <div className="flex h-96 items-center justify-center">No logs found</div>
+              )}
+            </>
```

Biome (1.9.4)

[error] Avoid using unnecessary Fragment.

A fragment is redundant if it contains only one child, or if it is the child of a html element, and is not a keyed fragment.
Unsafe fix: Remove the Fragment


(lint/complexity/noUselessFragments)